assets/_scss/elements/_table.scss

```diff
@@ -1,43 +1,40 @@
 table {
+  border-spacing: 0;
   margin: 2em 0;
   min-width: 100%;
-  
+
   thead {
+    tr {
+      background-color: $color-gray-lightest;
+    }
+
     th {
       text-align: left;
     }
   }
-  
+
   th, td {
-    border: 1px solid #757575;
+    border: 1px solid $color-gray;
     line-height: 1;
     padding: 1.5rem;
   }
 }
 
-.usa-table-bordered {
-  border-spacing: 0;
-
-  thead {
-    tr {
-      background-color: #eeeeee;
-    }
-  }
-}
-
 .usa-table-borderless {
```

Comment:
Do we like the class `-borderless` or prefer something else (for example clean, alt, minimal, subtle)?